packages/ddp-client/common/urlHelpers.js

```diff
@@ -1,18 +1,19 @@
-import { LivedataTest } from './namespace';
+import { LivedataTest } from './namespace.js';
 import { Random } from 'meteor/random';
 
 // XXX from Underscore.String (http://epeli.github.com/underscore.string/)
-var startsWith = function(str, starts) {
+function startsWith(str, starts) {
```

Comment:
I think we can just use `string.startsWith(prefix)` and string.endsWith(suffix)`, which is a `String` method that we polyfill.